pkgs/games/tuxtype/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, librsvg, SDL, SDL_image, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.3";
+  name = "tuxtype-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/tux4kids/tuxtype/archive/upstream/${version}.tar.gz";
+    sha256 = "0cv935ir14cd2c8bgsxxpi6id04f61170gslakmwhxn6r3pbw0lp";
+  };
+
+  patchPhase = ''
```

Comment:
use `postPatch` so that `patches` can still work.

Reply:
Thanks